content/md/what-is/what-is-an-attribute.md

```diff
@@ -5,63 +5,84 @@ title: What is an **attribute**?
 related: manage-your-attributes, what-is-an-attribute-group
 ---
 
-# Definition of an attribute
-
-An attribute is a product’s characteristic. Each product is composed of different attributes.
-
-Depending on your Akeneo Edition version, you can have up to 15 attribute types:
-- **identifier**: a unique code that identifies your product
-- **text** and **text area**: used for text data. A **text** attribute is **limited to 255 characters**, if you need more text data, use the **text area** attribute type.
-- **simple** or **multi select**
-- **reference data** (simple and multi select)
-- **boolean** (yes/no)
-- **date**
-- **number**
-- **price**
-- **metric** (with automatic conversions): this attribute type allows you to **automatically convert metric values into others to fit your export needs**. See the [Manage your Channels](manage-your-channels.html#create-a-channel) article.
-- **media file**: a simple file (pdf file for instance) or an image
-- **[asset collection](work-with-assets.html)**: digital resources like a video, picture, pdf file... _(Entreprise Edition only)_
-
-Attributes are gathered into [attribute groups](what-is-an-attribute-group.html) in Akeneo.
+# Introduction
+
+An attribute is a product’s characteristic. A product is usually composed of several attributes: an identification number, a name, a description, a price, a color... Depending on your Akeneo Edition version (Community or Enterprise), you can choose amongst **17 attribute types**. 
+
+In Akeneo, attributes are gathered into **families**, so all products belonging to the same family share the same attributes. If you are not familiar with the concept of families, check our [What is family?](what-is-a-family.html) article.
+
+You can easily create attribute groups to gather your attributes and better manage them. Read the following article to get more information about attribute groups: [What is an attribute group?](what-is-an-attribute-group.html).
+
+# Akeneo attribute types
+
+Here are the attribute types available in Akeneo:
+- **identifier** attribute: a code to identify your product, this code must be unique. It can be a SKU, a MPN... This attribute is mandatory.
+- **text** attribute: a single-line text field that can contain **up to 255 characters**, it is usually used for a product name.
+- **text area** attribute: a multi-line text field that can be used for a product description.
+- **simple** attribute: a single-choice select field coming with custom options. Only one value can be selected amongst the available options. 
+- **multi select** attribute: a multi-choice select field coming with custom options. More than one value can be selected amongst the available options.
+- **boolean** attribute: a yes/no attribute
```

Comment:
I do not want to be a nitpicker but in the UI it is called "Yes/No". Calling it here the exact same way as on the UI may ease the user understanding? (I think this article aims mostly beginners)